client/packages/common/src/ui/components/navigation/ListOptions/ListOptions.tsx

```diff
@@ -0,0 +1,81 @@
+import { CheckIcon, ChevronDownIcon } from '@common/icons';
+import {
+  List,
+  ListItemIcon,
+  ListItem,
+  ListItemText,
+  Divider,
+  Box,
+} from '@mui/material';
+import React from 'react';
+
+export type ListOptionValues = {
+  id: string;
+  value: string;
+};
+
+interface ListProps {
+  onClick: (id: string) => void;
+  options: ListOptionValues[];
+  currentId?: string;
+  enteredLineIds?: string[];
+}
+
+export const ListOptions = ({
+  onClick,
+  options,
+  currentId,
+  enteredLineIds,
+}: ListProps) => {
+  const startIcon = (
+    <CheckIcon
+      style={{
+        backgroundColor: '#33A901',
+        borderRadius: '50%',
+        padding: '2px',
+        color: 'white',
+        height: 18,
+        width: 18,
+      }}
+    />
+  );
+
+  const endIcon = (
+    <ChevronDownIcon
+      style={{ width: 17, height: 17, transform: 'rotate(-90deg)' }}
+    />
+  );
```

Comment:
I had them passed in as props at first since we might use different icons for different use cases, but can't think of them right now for what we will use it for. Easy to refactor later on if needed